### PR TITLE
Add new cluster type for OpenStack HW offload platform

### PR DESCRIPTION
**Pull request by EmilienM**
This new platform will let us test OCP on OpenStack using OVS HW
offload.

### CODE REVIEW

**Comment by EmilienM** [comment]
I had to rebase again.

File: pkg/api/types.go [hunk 1]
```diff
@@ -1368,6 +1372,8 @@ func (p ClusterProfile) LeaseType() string {
 		return "nutanix-quota-slice"
 	case ClusterProfileOpenStack:
 		return "openstack-quota-slice"
+	case ClusterProfileOpenStackHwoffload:
+		return "openstack-hwoffload-quota-slice"
```
Comment (droslean) [inline pkg/api/types.go:1376]: You will have to create that slice in order to use this cluster profile in a job.

Comment (EmilienM) [reply pkg/api/types.go:1376]: done here https://github.com/openshift/release/pull/2750

**Pull request merged**